docs/_includes/icon.html

```diff
@@ -90,4 +90,21 @@ <h3 id="icon-disabled">Disabled Icons</h3>
         <use href="#icon-chevron-right-24"></use>
     </svg>
     {% endhighlight %}
+
+    <h3>Deprecated Icons</h3>
+    <p>These deprecated icons have either been removed or renamed. We keep deprecated icon files around to allow some time for transitioning to new icons. Deprecated icon files and mentions in docs, however, get removed with the following major Skin versions.</p>
+
+    <section class="section-notice section-notice--attention" role="region" aria-label="Attention" aria-roledescription="Notice">
+        <div class="section-notice__header" id="section-notice-attention">
+            <svg class="icon icon--attention-filled-16" focusable="false" height="16" width="16" aria-label="Attention">
+                {% include symbol.html name="attention-filled-16" %}
+            </svg>
+        </div>
+        <div class="section-notice__main">
+            <h3 class="section-notice__title">Do NOT Use These!</h3>
+            <p>The following icons have been deprecated. They will be deleted in the next major Skin version. To transition usage to the new icons, please refer to the icons above and/or refer to the icon transition spreadsheet.</p>
```

Comment:
Is the icon transition spreadsheet also open source? If it is, we should link to it here.

Reply:
No, I didn't include that because it is internal.

Reply:
If it's internal I don't think we should mention it here. Maybe we should just say "please use equivalent icons in the section above"

Reply:
I don't think there's any danger in simply mentioning a hypothetical spreadsheet that may or may not exist. :-) A link would not be prudent, however.